Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs

```diff
@@ -576,3 +591,103 @@ getChargesBreakup paymentBreakUpArr = map (\(PaymentBreakUp item) -> {val : "₹
+                                                                                                   imageUrl = "ny_ic_locomotor_pickup" <> (getAssetStoreLink FunctionCall) <> "ny_ic_locomotor_pickup.png"
+                                                                                                  }
+                                      Just ST.OTHER_DISABILITY, true -> accessibilityConfig{ primaryText = getString CUSTOMER_MAY_NEED_ASSISTANCE, 
+                                                                                             secondaryText = getString CUSTOMER_HAS_DISABILITY_PLEASE_ASSIST_THEM,
+                                                                                             imageUrl = "ny_ic_disability_illustration" <> (getAssetStoreLink FunctionCall) <> "ny_ic_disability_illustration.png"
+                                                                                            }   
+                                      Just ST.OTHER_DISABILITY, false ->  accessibilityConfig{ primaryText = getString CUSTOMER_MAY_NEED_ASSISTANCE,
+                                                                                               secondaryText = getString CUSTOMER_HAS_DISABILITY_PLEASE_ASSIST_THEM,
+                                                                                               imageUrl = "ny_ic_disability_illustration" <> (getAssetStoreLink FunctionCall) <> "ny_ic_disability_illustration.png"
+                                                                                              }  
+                                      _ , _-> accessibilityConfig 
+
+
+
+getAccessibilityHeaderText :: ST.HomeScreenState -> {primaryText :: String, secondaryText :: String, imageUrl :: String}
+getAccessibilityHeaderText state = if state.data.activeRide.status == INPROGRESS then 
+                        case state.data.activeRide.disabilityTag of   
+                          Just ST.HEAR_IMPAIRMENT -> {primaryText : getString CUSTOMER_HAS_HEARING_IMPAIRMENT, secondaryText : getString PLEASE_HELP_THEM_AS_YOU_CAN, imageUrl : "ny_ic_poor_hearing" <> (getAssetStoreLink FunctionCall) <> "ny_ic_poor_hearing.png"} 
+                          Just ST.BLIND_AND_LOW_VISION -> {primaryText : getString CUSTOMER_HAS_LOW_VISION, secondaryText : getString PLEASE_HELP_THEM_AS_YOU_CAN, imageUrl : "ic_accessibility_vision" <> (getAssetStoreLink FunctionCall) <> "ic_accessibility_vision.png"}
+                          Just ST.LOCOMOTOR_DISABILITY -> {primaryText : getString CUSTOMER_HAS_LOW_MOBILITY, secondaryText : getString PLEASE_HELP_THEM_AS_YOU_CAN,  imageUrl : "ny_ic_disability_purple" <> (getAssetStoreLink FunctionCall) <> "ny_ic_disability_purple.png"}
+                          Just ST.OTHER_DISABILITY -> {primaryText : getString CUSTOMER_HAS_DISABILITY, secondaryText : getString PLEASE_HELP_THEM_AS_YOU_CAN, imageUrl : "ny_ic_disability_purple" <> (getAssetStoreLink FunctionCall) <> "ny_ic_disability_purple.png"}
```

Comment:
create a default value and try to modify the data from that. and last two cases are same try to combine them.